pipeline/brew/brew.go

```diff
@@ -85,7 +85,7 @@ func (Pipe) Description() string {
 
 // Run the pipe
 func (Pipe) Run(ctx *context.Context) error {
-	if ctx.Config.Brew.Repo == "" {
+	if ctx.Config.Brew.Repo == "" || ctx.Config.Brew.Repo == "none" {
```

Comment:
Does it even need the `"none"` behavior or can you directly set it to `""` in the YAML config?
I haven't tried it though.